deployer/src/handlers/mod.rs

```diff
@@ -795,6 +794,7 @@ pub async fn get_secrets(
     }
 }
 
+#[instrument(skip_all, fields(%project_name))]
```

Comment:
Fyi, `project_name` is always automatically traced for all deployer routes currently.

https://github.com/shuttle-hq/shuttle/blob/ffb760abb92f644cde9c777b6973e412b46c8ebe/deployer/src/handlers/mod.rs#L215-L221

Reply:
Good to know! It was mainly for consistency in relation to the others.